lib/services/cups.pm

```diff
@@ -81,6 +81,16 @@ sub check_function {
     record_info "access_log", "Access log should containt successful job submits";
     assert_script_run 'grep "Send-Document successful-ok" /var/log/cups/access_log';
 
+    # Check error log
+    my $error_log = "/var/log/cups/error_log";
+
+    if (script_run("test -s $error_log") == 0) {
+        record_info("error log");
+        upload_logs($error_log, failok => 1);
+
+        assert_script_run('! grep -q "Unrecoverable error" ' . "$error_log");
```

Comment:
I'm just curious how much do you care about `Unrecoverable error` ?
It is intended that the test will fail here ?

Reply:
Based on https://progress.opensuse.org/issues/58370 it was a bit of a big deal, as not catching it caused a regression from a ghostscript update. The customer was not pleased and it was demanded to test this in openQA in order to avoid such a case in the future.